File: source/src/main/java/ro/fii/licenta/api/dto/MemberDTO.java
package ro.fii.licenta.api.dto;

public class MemberDTO {
	private Long id;

	private UserDTO user;

	private NgoDTO ngo;

	private NgoFunctionDTO function;

	private OrganizationalComponentDTO organizationalComponent;

	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public UserDTO getUser() {
		return user;
	}

	public void setUser(UserDTO user) {
		this.user = user;
	}

	public NgoDTO getNgo() {
		return ngo;
	}

	public void setNgo(NgoDTO ngo) {
		this.ngo = ngo;
	}

	public NgoFunctionDTO getFunction() {
		return function;
	}

	public void setFunction(NgoFunctionDTO function) {
		this.function = function;
	}

	public OrganizationalComponentDTO getOrganizationalComponent() {
		return organizationalComponent;
	}

	public void setOrganizationalComponent(OrganizationalComponentDTO organizationalComponent) {
		this.organizationalComponent = organizationalComponent;
	}
}

File: frontend/src/app/shared/dto/MemberDTO.ts
import {UserDTO} from './UserDTO';
import {NgoDTO} from './NgoDTO';
import {FunctionDTO} from './FunctionDTO';
import {OrganizationalComponentDTO} from './OrganizationalComponentDTO';

export class MemberDTO {
  public id;
  public user: UserDTO;
  public ngo: NgoDTO;
  public function: FunctionDTO;
  public organizationalComponent: OrganizationalComponentDTO;
}
